stix_shifter/stix_translation/src/modules/guardium/stix_to_query.py

```diff
@@ -0,0 +1,36 @@
+import logging
+
+#from ...patterns.parser import generate_query
+from ..base.base_query_translator import BaseQueryTranslator
+#from . import data_mapping
+from . import query_constructor
+import logging
```

Comment:
You are already importing logging at the top.

Reply:
Removed.